.github/container-tests-al2.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -x
```

Comment:
Should this be `set -ex`?

Reply:
It's not strictly necessary, as `rpmbuild` is the last command of the script, so the script will exit with its exit code. However, in the interest of avoiding a situation where future changes break this assumption, I'll at `-e`